### PR TITLE
Black-ed setup.py and added .DS_Store to gitignore

### PR DESCRIPTION
**Pull request by lockefox**
Updated setup.py to play nice with black https://github.com/psf/black

Highly suggest adding black to the make lint steps as well

### CODE REVIEW

**Review by patdaburu — APPROVED** [review]
These changes look good to me.

File: {{cookiecutter.project_name}}/.gitignore [hunk 1]
```diff
@@ -104,3 +104,6 @@ ENV/
 
 # JetBrains
 .idea
+
```
Comment (vladdoster) [inline {{cookiecutter.project_name}}/.gitignore:107]: Please add

```
# VS Code settings
.vscode
```

**Review by vladdoster — CHANGES_REQUESTED** [review]
Please see my comment. Thank you! 🚀

**Comment by lockefox** [comment]
added requested vscode block to .gitignore

**Review by vladdoster — APPROVED** [review]
Awesome work @lockefox ! I am a big fan of black!

**Pull request merged**